Objects/object.c

```diff
@@ -2491,6 +2491,24 @@ _Py_ResurrectReference(PyObject *op)
 
 
 #ifdef Py_TRACE_REFS
+void
+_Py_NormalizeImmortalReference(PyObject *op)
```

Comment:
Having some comment here would be helpful.  Otherwise this function looks quite mysterious.

Reply:
done!